pycheribuild/config/target_info.py

```diff
@@ -543,7 +546,8 @@ def is_mips(self, include_purecap: bool = None):
         return self._check_arch(CPUArchitecture.MIPS64, include_purecap)
 
     def is_riscv(self, include_purecap: bool = None):
-        return self._check_arch(CPUArchitecture.RISCV64, include_purecap)
+        return self._check_arch(CPUArchitecture.RISCV64, include_purecap) or \
+            self._check_arch(CPUArchitecture.RISCV32, include_purecap)
```

Comment:
Might be nicer to make `_check_arch` take a list instead.

Reply:
I think this is fine since there are no other cases that need a list.